lib/sidekiq_adhoc_job/web/assets/javascript/require_confirm.js

```diff
@@ -0,0 +1,11 @@
+
+document.getElementById('adhoc-jobs-submit-form').addEventListener('submit', (event) => {
+  const confirmPrompt = document.getElementById('presented-job').getAttribute('data-confirm-prompt');
```

Comment:
so we create another `<div>` because we can't extrapolate the ERB `<%= @presented_job.confirm_prompt_message %>` into javascript is that correct?

Reply:
Yup, that `<div>` is to pass the prompt message into the javascript file. Not sure if there is a more elegant way to do it.

Reply:
haha no worries I think this works well, just wanted to confirm my understanding. Thanks!